test/notifier_test.exs

```diff
@@ -22,6 +22,11 @@ defmodule NotifierTest do
 
       send(self(), %{exception: %{subject: subject, body: body}})
     end
+
+    @impl BoomNotifier.Notifier
```

Comment:
I think this is not needed since we are using the `MissingParameterNotifier` to check the parameters validation